docs/manuals/core/versioning/production_mode.md

```diff
@@ -18,76 +18,217 @@ To convert an experiment version to a production, you can run the Taipy applicat
 ```
 
-After running the commands above, version 2.0 is an experiment version again. It is no longer a
+After running the commands above, version 0.1 is an experiment version again. It is no longer a
 production version.
+
+# Change config of an existing production version
+
+Similar to experiment mode, to re-run an existing production version, the configuration
+of the application must not have any breaking conflict.
+
+Let's assume multiple changes have been made to the configuration in `main.py`.
+
+```python linenums="1" title="main.py"
+{%
+include-markdown "./code_example/main_with_multiple_changes.py"
+comments=false
+%}
+```
+
+```console
+$ python main.py --production 1.0
+[2023-01-25 12:52:05,484][Taipy][ERROR] The version 1.0 Configuration is conflicted with the current Configuration:
+    DATA_NODE "input" has attribute "path" added: input.pkl
+    DATA_NODE "output" has attribute "path" added: output.pkl
+    DATA_NODE "input" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    DATA_NODE "output" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    TASK "example_algorithm" has attribute "skippable" modified: False:bool -> True:bool
+
+Please add a new production version with migration functions or run your application with --taipy-force option to override the Config of production version 1.0.
+```
+
+In the example above, when re-running production version 1.0, Taipy detects and displays all the
+changes. As shown in the message, there are 2 options to deal with these changes.
+
+First, you can run the production version with the `--taipy-force` option to force Taipy to update
+the configuration of the version before re-running the application.
+
+```console
+$ python main.py --production 1.0 --taipy-force
+[2023-07-04 10:25:41][Taipy][ERROR] The version 1.0 Configuration is conflicted with the current Configuration:
+    DATA_NODE "input" has attribute "path" added: input.pkl
+    DATA_NODE "output" has attribute "path" added: output.pkl
+    DATA_NODE "input" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    DATA_NODE "output" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    TASK "example_algorithm" has attribute "skippable" modified: False:bool -> True:bool
+[2023-07-04 10:25:41][Taipy][WARNING] Option --taipy-force is detected, overriding the configuration of version 1.0 ...
+[2023-07-04 10:25:41][Taipy][INFO] Version 1.0 is already a production version.
+[2023-07-04 10:25:41][Taipy][INFO] job JOB_example_algorithm_7a54227c-159d-4768-99c3-8c19c84a2e61 is completed.
+```
+
+As you can see, the application is run successfully after updating the configuration.
+
+!!! Warning
+
+    By forcing the configuration update, you must be aware that old entities instantiated before
+    the configuration change may not be compatible.
+
+Second, to avoid overriding the Configuration of that production version, you can create a new
```

Comment:
"Configuration" why a capital?

Reply:
This probably a systematic error. Let me fix those